pwiz_tools/Skyline/Test/CodeInspectionTest.cs

```diff
@@ -420,7 +421,20 @@ void InspectTutorialAuditLogs(string root, List<string> errors)
                 foreach (var formType in formTypes)
                 {
                     // Now find all forms that inherit from formType
-                    var assembly = formType == typeof(Form) ? Assembly.GetAssembly(typeof(FormEx)) : Assembly.GetAssembly(formType);
+                    // Search for forms in the same assembly as the base class with a few exceptions
+                    Assembly assembly;
+                    if (formType == typeof(Form))
+                    {
+                        assembly = typeof(SkylineWindow).Assembly;
+                    }
+                    else if (formType == typeof(CommonFormEx))
+                    {
+                        assembly = typeof(ViewEditor).Assembly;
```

Comment:
Thanks for fixing this.  I just finished debugging to try to understand why the test was failing, and figured that since CommonFormEx was moved to pwiz.CommonUtil, none of the forms in pwiz.Common were getting "discovered" in FindForms().  Since we don't have a base form type in pwiz.Common anymore, I got around it by doing this:
`assembly = formType == typeof(CommonFormEx) ? Assembly.GetAssembly(typeof(PivotEditor)) : assembly;`
Looks like you did something similar.  Thanks!